lib/winapi/debug.c

```diff
@@ -1,8 +1,14 @@
-#include <winbase.h>
+#include <windows.h>
 #include <winnt.h>
 
 #include <string.h>
 
+BOOL IsDebuggerPresent (VOID)
+{
+    // FIXME: Can we detect debugger presence on the Xbox?
+    return FALSE;
```

Comment:
There's `KdDebuggerEnabled` / `KdDebuggerNotPresent`.

XBDM also sets up some hooks into the KPCR or something. I think that's where it communicates performance counters - which [even work for retail games, as shown in my FPS overlay](https://github.com/JayFoxRox/xbox-fps-overlay).

Obviously this is something we should be discussing in a follow-up PR / issue, not here.

Reply:
We should create an issue about this (marking it as a stub, and discussing the possibility / implementation options)